src/features/call/components/Survey/SurveyText.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC, useState } from 'react';
+import { Box, FormControl, FormLabel, TextField } from '@mui/material';
+
+import messageIds from 'features/surveys/l10n/messageIds';
+import { useMessages } from 'core/i18n';
+import { ZetkinSurveyTextQuestionElement } from 'utils/types/zetkin';
+import SurveyContainer from 'features/surveys/components/surveyForm/SurveyContainer';
+import SurveySubheading from 'features/surveys/components/surveyForm/SurveySubheading';
+import SurveyQuestionDescription from 'features/surveys/components/surveyForm/SurveyQuestionDescription';
+
+export type SurveyOptionsQuestionProps = {
+  element: ZetkinSurveyTextQuestionElement;
+  onChange: (value: string) => void; // Added prop for parent state update
+};
+
+const SurveyText: FC<SurveyOptionsQuestionProps> = ({ element, onChange }) => {
```

Comment:
I don't think we need a new component for this. We already have `SurveyTextQuestion`. Correct me if I'm missing something and this needs to be duplicated for some reason.

Reply:
Correct. Now that I'm using `<SurveyElements />` I don't it anymore. I'll delete it.